frontend/test/__support__/cypress.js

```diff
@@ -3,49 +3,74 @@ import "./commands";
 
 export const version = require("../../../version.json");
 
+export const USER_GROUPS = {
+  ALL_USERS_GROUP: 1,
+  ADMIN_GROUP: 2,
+  COLLECTION_GROUP: 4,
+  DATA_GROUP: 5,
+};
+
+const { ALL_USERS_GROUP, COLLECTION_GROUP, DATA_GROUP } = USER_GROUPS;
+
 export const USERS = {
   admin: {
     first_name: "Bobby",
     last_name: "Tables",
-    username: "admin@metabase.com",
+    email: "admin@metabase.com",
     password: "12341234",
   },
   normal: {
     first_name: "Robert",
     last_name: "Tableton",
-    username: "normal@metabase.com",
+    email: "normal@metabase.com",
     password: "12341234",
+    group_ids: [ALL_USERS_GROUP, COLLECTION_GROUP, DATA_GROUP],
   },
   nodata: {
     first_name: "No Data",
     last_name: "Tableton",
-    username: "nodata@metabase.com",
+    email: "nodata@metabase.com",
     password: "12341234",
+    group_ids: [ALL_USERS_GROUP, COLLECTION_GROUP],
   },
   nocollection: {
     first_name: "No Collection",
     last_name: "Tableton",
-    username: "nocollection@metabase.com",
+    email: "nocollection@metabase.com",
     password: "12341234",
+    group_ids: [ALL_USERS_GROUP, DATA_GROUP],
   },
   none: {
     first_name: "None",
     last_name: "Tableton",
-    username: "none@metabase.com",
+    email: "none@metabase.com",
     password: "12341234",
+    group_ids: [ALL_USERS_GROUP],
+  },
+  sandboxed: {
+    first_name: "User",
+    last_name: "1",
+    email: "u1@metabase.test",
+    password: "12341234",
+    login_attributes: {
+      user_id: "1",
+      user_cat: "Widget",
+    },
+    group_ids: [ALL_USERS_GROUP, COLLECTION_GROUP],
   },
 };
 
-export const USER_GROUPS = {
-  ALL_USERS_GROUP: 1,
-  ADMIN_GROUP: 2,
-  COLLECTION_GROUP: 4,
-  DATA_GROUP: 5,
-};
+export function createUser(user = {}) {
```

Comment:
I wouldn't add a default for `user` since it's required. We'll never use `{}`.

Reply:
Oh, ok. I'm usually adding this for the implicit type inferring. It doesn't have a direct value before we switch to Typescript, but I think VSCode picks that up.

I can delete it if it's confusing, although the request with `{}` would fail anyway.

Reply:
Huh interesting I didn't realize VSCode does that. It seems weird to me to add unused code purely for type inference, but it's probably fine. An alternative is to continue using Flow type annotations since apparently [those will continue to work](https://github.com/metabase/metabase/pull/14236#issuecomment-790875483).

Reply:
I simplified this function, the same way you created `signIn()` that accepts the string, rather than the full user object. That way, we don't have to import `USERS` and destructure every possible user from it. Less imports, cleaner code. Win-win :)